src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -146,10 +146,10 @@ bool KeyboardShortcuts::Save()
 void KeyboardShortcuts::Set(int32_t key)
 {
     // Unmap shortcut that already uses this key
-    int32_t shortcut = GetFromKey(key);
-    if (shortcut != SHORTCUT_UNDEFINED)
+    auto shortcut = GetFromKey(key);
+    if (shortcut != static_cast<int32_t>(Input::Shortcut::Undefined))
     {
-        _keys[shortcut] = SHORTCUT_UNDEFINED;
+        _keys[shortcut] = static_cast<int32_t>(Input::Shortcut::Undefined);
```

Comment:
this is not Input::Shortcut::Undefined but a different variable with the same value.